### PR TITLE
S2t3

### PR DESCRIPTION
**Pull request by isabellecochrane**

### CODE REVIEW

**Review by dotNATE — COMMENTED** [review]
Only 1 comment, otherwise seems all good! (taking into account the messages re docblocks and hinting etc)

File: src/Classes/Api.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    private array $allListings;
+    private array $lettingTypes;
+    private array $lettingStatuses;
+
+    /**
+     * Api constructor.
+     */
+    public function __construct() {
+        $this->request = curl_init();
+        var_dump($this->request);
```
Comment (dotNATE) [inline src/Classes/Api.php:17]: Is this var_dump functional or can it go?

File: src/Classes/Api.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<?php
+namespace ListingsApp\Classes;
+/**
+ * send api request to receive listings data
+ */
```
Comment (dotNATE) [inline src/Classes/Api.php:5]: Don't need a doc block for a class. Unless we do, in which case please let me know so we can doc block ours!

**Review by dotNATE — COMMENTED** [review]
All seems mos

File: src/Classes/Api.php (hunk 1 above)
Comment (isabellecochrane) [reply src/Classes/Api.php:17]: it can go, thx

File: src/Classes/Api.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?php
+namespace ListingsApp\Classes;
+
+class Api {
+    private $request;
+    private $allListings;
+    private $lettingTypes;
+    private $lettingStatuses;
```
[the remark this answers is not in the document]
Comment (isabellecochrane) [reply src/Classes/Api.php]: private we think, you only need to access the actual properties in the class because the returns can be accessed else where

**Pull request merged**